src/osm/GeometryHandler.cpp

```diff
@@ -61,137 +79,402 @@ osm2rdf::osm::GeometryHandler<W>::GeometryHandler(
 
 // ___________________________________________________________________________
 template <typename W>
-osm2rdf::osm::GeometryHandler<W>::~GeometryHandler() = default;
-
-// ____________________________________________________________________________
-template <typename W>
-std::string osm2rdf::osm::GeometryHandler<W>::statisticLine(
-    std::string_view function, std::string_view part, std::string_view check,
-    uint64_t outerId, std::string_view outerType, uint64_t innerId,
-    std::string_view innerType, std::chrono::nanoseconds durationNS,
-    bool result) {
-  return "{"
-         "\"function\":\"" +
-         std::string(function) +
-         "\","
-         "\"part\":\"" +
-         std::string(part) +
-         "\","
-         "\"check\":\"" +
-         std::string(check) +
-         "\","
-         "\"outer_id\":" +
-         std::to_string(outerId) +
-         ","
-         "\"outer_type\":\"" +
-         std::string(outerType) +
-         "\","
-         "\"inner_id\":" +
-         std::to_string(innerId) +
-         ","
-         "\"inner_type\":\"" +
-         std::string(innerType) +
-         "\","
-         "\"duration_ns\":" +
-         std::to_string(durationNS.count()) +
-         ","
-         "\"result\":" +
-         (result ? "true" : "false") +
-         ""
-         "},\n";
-}
-
-// ____________________________________________________________________________
-template <typename W>
-void osm2rdf::osm::GeometryHandler<W>::area(const osm2rdf::osm::Area& area) {
-  auto geom = area.geom();
-  if (_config.simplifyGeometries > 0) {
-    geom = simplifyGeometry(geom);
+GeometryHandler<W>::~GeometryHandler() = default;
+
+// ____________________________________________________________________________
+template <typename W>
+void GeometryHandler<W>::relation(const Relation& rel) {
+  // careful, tags is returned via copy (why?)
```

Comment:
This was changed in in Way.h and should be changeable in Relation.h too. https://github.com/ad-freiburg/osm2rdf/blob/095ef8e6f0d9e46490232d7b4b5c5d9bc7117b73/include/osm2rdf/osm/Way.h#L50

Reply:
I think this is already changed in Relation.h -> maybe this comment could be removed?

Reply:
Indeed!